integrations/oracles/band-protocol.md

```diff
@@ -49,15 +49,15 @@ struct ReferenceData {
 }
 ```
 
-The second function, `getReferenceDataBulk`, takes information as data arrays. For example, if we pass in `['BTC','BTC','ETH']` as base and `['USD','ETH','EUR']` as quote, the `ReferenceData`returned array contains the information regarding the following pairs:
+The second function, **getReferenceDataBulk**(*string[]* _bases, *string[]* _quotes), takes information as data arrays. For example, if we pass in `['BTC','BTC','ETH']` as base and `['USD','ETH','EUR']` as quote, the `ReferenceData`returned array contains the information regarding the following pairs:
```

Comment:
Here I would leave the function name as it was. I would only change it  in bullets lists and we have to be consistent with that formatting